model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -92,13 +92,11 @@ class FindingsMatcher(
             "The given license and copyright findings must all point to the same file."
         }
 
-        val allCopyrightStatements = copyrights.toSet()
-
         // If there is no license finding but copyright findings, associate them with all root licenses.
-        if (licenses.isEmpty()) return rootLicenses.associateBy({ it }, { allCopyrightStatements })
+        if (licenses.isEmpty()) return rootLicenses.associateBy({ it }, { copyrights.toSet() })
```

Comment:
Would you mind to briefly explain in the commit message why this makes sense here? E.g. something like "Because the variable is only used once in either code path, and the conversion to a set should not be too expensive."

Reply:
I'd see this rather as common sense and thus prefer to omit it. But as you ask for it I'll just add it.

Reply:
Well, at a quick glance it could seem as if the variable is used twice in the *same* code path if you overlook the `return` statements, and if that was that case I personally would not have inlined the variable. Hence I believe an explanation is justified here.

Reply:
done.